instruments/keithley/keithley580.py

```diff
@@ -262,8 +259,11 @@ def input_range(self):
 
         :type: `~quantities.quantity.Quantity` or `str`
         """
-        value = float(self.parse_status_word(self.get_status_word())['range'])
-        return value * u.ohm
+        value = self.parse_status_word(self.get_status_word())['range']
```

Comment:
The input range can also be set to 'auto', in that case, parsing the status word will return 'auto'. This so far resulted in a `ValueError`, since 'auto cannot be converted to float. Catching that case now. This also makes the getter consistent with the setter, which accepts `... = 'auto'` as an argument.